Source/objects.cpp

```diff
@@ -4325,6 +4330,11 @@ void SyncL3Doors(Object &door)
 
 } // namespace
 
+unsigned int Object::GetId() const
+{
+	return abs(dObject[position.x][position.y]) - 1;

```

Comment:
Uh, interesting solution. I think it works here for now, but probably will need a change in the long run.

Reply:
Definitely. This is ... something that works for now.

I'd much rather write a property when the object is created as this allows creating objects from the level deltas based on what the host has generated, instead of having to have the object generation perfectly synced. But that requires going through the way objects are created to ensure we don't blow away any attributes before they get fully initialised.